### PR TITLE
Correct the error message for incorrect syntax in the children of a text node within a conversation

### PR DESCRIPTION
**Pull request by warp-core**
**Bug fix**

Thanks to acquire_squids on Disocrd for reporting this.

## Summary
The correct syntax is `to display`, but the error message says `to show`.
This PR updates the message to describe the correct syntax.

## Testing Done
No.

### CODE REVIEW

**Review by mOctave — APPROVED** [review]

**Comment by TomGoodIdea** [comment]
The comment on this function has the same problem.

File: source/Conversation.cpp [hunk 1]
```diff
@@ -555,7 +555,7 @@ bool Conversation::ElementIsValid(int node, int element) const
 
 
 // Parse the children of the given node to see if then contain any "gotos," or
-// "to shows." If so, link them up properly. Return true if gotos or
+// "to displays." If so, link them up properly. Return true if gotos or
```
Comment (mOctave) [inline source/Conversation.cpp:558]: Tangential to this PR, but punctuation and `s`s might benefit from being outside the quotes for readability. Or, if it was written `"goto" or "to display" nodes`.

Comment (warp-core) [inline source/Conversation.cpp:558]:
```suggestion
// Parse the children of the given node to see if then contain any "goto" or
// "to display" nodes. If so, link them up properly. Return true if gotos or
```

File: source/Conversation.cpp [hunk 2]
```diff
@@ -555,7 +555,7 @@ bool Conversation::ElementIsValid(int node, int element) const
 
 
 // Parse the children of the given node to see if then contain any "gotos," or
```
Comment (warp-core) [inline source/Conversation.cpp:557]:
```suggestion
// Parse the children of the given node to see if then contain any "goto" or
```

File: source/Conversation.cpp (hunk 1 above)
Comment (warp-core) [inline source/Conversation.cpp:558]:
```suggestion
// "to display" nodes. If so, link them up properly. Return true if gotos or
```

**Review by TomGoodIdea — APPROVED** [review]

**Review by TheGiraffe3 — APPROVED** [review]

File: source/Conversation.cpp (hunk 1 above)
Comment (warp-core) [reply source/Conversation.cpp]: Applied the second option.

**Review by mOctave — APPROVED** [review]

**Pull request merged**